samples/SignalRSamples/Hubs/Streaming.cs

```diff
@@ -1,10 +1,10 @@
 using System;
```

Comment:
What are the rules on copyright headers in samples? Some files have them and others don't.

Reply:
This seems like as good a time as any to get them consistent. They should all have headers. I'll review them